client/my-sites/stats/post-performance/index.jsx

```diff
@@ -189,9 +129,12 @@ export default connect( ( state, ownProps ) => {
 	const query = { status: 'published' };
 	const posts = site ? getSitePostsForQuery( state, site.ID, query ) : null;
 	const post = posts && posts.length ? posts[ 0 ] : null;
+	const viewCount = post && site ? getPostStat( state, 'views', site.ID, post.ID ) : null;
+
 	return {
+		viewCount,
 		query,
 		post,
-		loading: isRequestingSitePostsForQuery( state, site.ID, query )
+		loading: ! site || ! posts || viewCount === null
```

Comment:
It looks like this naming convention is in place in a number of other components, so don't let this block you.